views/dist/back.js.LICENSE.txt

```diff
@@ -0,0 +1,18 @@
+/**
```

Comment:
Do we need to version this file?

Reply:
It is unneeded if `npm run build` is run automatically in workflows.